pkg/config/plugins/runtime/k8s/config.go

```diff
@@ -36,29 +76,274 @@ type AdmissionServerConfig struct {
 	CertDir string `yaml:"certDir" envconfig:"kuma_kubernetes_admission_server_cert_dir"`
 }
 
+// Injector defines configuration of a Kuma Sidecar Injector.
+type Injector struct {
+	// SidecarContainer defines configuration of the Kuma sidecar container.
+	SidecarContainer SidecarContainer `yaml:"sidecarContainer,omitempty"`
+	// InitContainer defines configuration of the Kuma init container.
+	InitContainer InitContainer `yaml:"initContainer,omitempty"`
+	// CNIEnabled if true runs kuma-cp in CNI compatible mode
+	CNIEnabled bool `yaml:"cniEnabled" envconfig:"kuma_kubernetes_injector_cni_enabled"`
```

Comment:
I believe the path is `runtime.kubernetes.injector.CNIEnabled` which should be `kuma_runtime_kubernetes_injector_cni_neabled`

Reply:
I checked env names in `AdmissionServerConfig`. They have the same nesting level as `CNIEnabled`:
- `runtime.kubernetes.injector.CNIEnabled` and
- `runtime.kubernetes.admissionServer.address`

And `address` has the name - `kuma_kubernetes_admission_server_address` (other fields as well).

So I assumed that we should use only the last 2 levels to avoid overcomplicating names.

Reply:
This is a mistake then. The idea was that env vars are 1:1 with yaml path. Can you fix it?